_posts/code/PerfContest/4/canny-run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o xtrace
+
+wget --no-clobber https://johnysswlab.com/downloads/221575.tar.gz
+tar -xf 221575.tar.gz
+
+CC=clang
```

Comment:
I think we will have many participants that use Microsoft compiler on Windows. :)
Since this is a single-file benchmark, writing a Cmake file for it may be overkill. But please mention in the announcement post that they need to create a VS project and use corresponding MSVC compilation options.

Reply:
I have no idea how to do it, last time I worked on Windows was 10 years ago :(

Reply:
I see. I'm not a frequent user of Windows either, but many people still are. They could use clang with MS visual studio as well, but still the default for them is MSVC compiler.